src/exchange/account/alias/account-alias.controller.js

```diff
@@ -11,36 +11,41 @@ angular
                 messaging,
                 translator
             };
-
             this.$routerParams = Exchange.getParams();
-            this.aliasLoading = false;
+            this.getAliasesParams = {};
```

Comment:
get should should be reserve as an instruction for function only, not for public variables